src/Components/AllMovies/AllMovies.js

```diff
@@ -1,24 +1,26 @@
-import './AllMovies.css'
-import MovieCard from '../MovieCards/MovieCards'
+import "./AllMovies.css";
+import MovieCard from "../MovieCards/MovieCards";
+import PropTypes from "prop-types";
 
-function AllMovies({moviesData, viewMovieBlowUp}) {
+function AllMovies({ moviesData, viewMovieBlowUp }) {
   const movieCards = moviesData.map((movie) => {
     return (
-      <MovieCard 
-        imagePath={movie.poster_path} 
+      <MovieCard
+        imagePath={movie.poster_path}
         rating={movie.average_rating}
         title={movie.title}
         id={movie.id}
         key={movie.id}
         viewMovieBlowUp={viewMovieBlowUp}
       />
-    )
-  })
-  return (
-    <div className="movie-container">
-    {movieCards}
-    </div>
-  )
+    );
+  });
+  return <div className="movie-container">{movieCards}</div>;
 }
 
-export default AllMovies
\ No newline at end of file
+AllMovies.propTypes = {
+  moviesData: PropTypes.array.isRequired,
```

Comment:
This is a good start on PropTypes but doesn't dig quite deep enough.  Can you go further to specify and enforce that the contents of the array are what you expect?  Should it be an array of strings? Numbers? Objects?